.prettierignore

```diff
@@ -6,3 +6,4 @@ certs
 public
 .cache
 .next
+/out
```

Comment:
I think this needs to be `out/` or just `out` vs. having it in the root

Reply:
Noted, should be changed now